pkgs/network/src/interface.zig

```diff
@@ -107,7 +110,7 @@ pub const GenericGossipHandler = struct {
 
             const publishWrapper = try self.allocator.create(MessagePublishWrapper);
             publishWrapper.* = MessagePublishWrapper{ .handler = handler, .data = data, .networkId = self.networkId };
-            std.debug.print("\n\n\nnetwork-{d}:: schedueling ongossip publishWrapper={any} on loop for topic {any}\n", .{ self.networkId, topic, publishWrapper });
+            std.debug.print("\n\n\nnetwork-{d}:: schedueling ongossip publishWrapper={any} on loop for topic {any}\n\n", .{ self.networkId, topic, publishWrapper });
```

Comment:
@noopur23 add the following three tasks to your log EPIC task:

1. pass same logger to network which you pass to node (in case of mock network, create and pass its own logger with default scope) and change std.debug statements to the logger.debug ones
2. add logger to validator and pass node logger to validator as well and replace std.debug there as well
3. colore code the level (like [info] [debug]) into different colors, as well as timestamp and where we print the scope